l10n_it_ricevute_bancarie/views/partner_view.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<odoo>
+
+    <!--========================================================-->
+    <!--                    RES.PARTNER - FORM                    -->
+    <!--========================================================-->
+    <record id="view_partner_form_riba" model="ir.ui.view">
+        <field name="name">res.parner.form.riba</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="groups_id" eval="[(4, ref('account.group_account_user'))]"/>
+        <field name="arch" type="xml">
+            <field name="property_payment_term_id" position="after">
```

Comment:
questo separatore ed il gruppo rompono la vista, facendo sparire tutte le label ai campi.
oltretutto il campo property_payment_term_id non deriva dal modulo base, ma dal modulo account

Reply:
@ruben-tonetto puoi  sistemare questa cosa?

Reply:
@labaggio purtroppo mi serve parecchio tempo per sistemarlo; meglio che ti passo il modulo se tu riesci a completarlo in tempi brevi

Reply:
Ho fatto un PR sul tuo repo, devi solo fare il merge
[lbaggio#2](https://github.com/ruben-tonetto/l10n-italy/pull/2)